independent-publisher-connectors/FreeAgent/apiProperties.json

```diff
@@ -0,0 +1,82 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "Ig_9msmfd2ua93ZVxeO65w",
+          "scopes": [
+

```

Comment:
@nk-gears Are you sure that there are no scopes required for creating a new connection. If not, please remove spaces in scopes values.

` "scopes": []`

Reply:
@sriyen-msft Fixed and Committed.